packages/compass-e2e-tests/tests/auto-update.test.ts

```diff
@@ -0,0 +1,67 @@
+import { expect } from 'chai';
+import {
+  init,
+  cleanup,
+  Selectors,
+  screenshotPathName,
+} from '../helpers/compass';
+
+describe('Auto-update', function () {
+  it('auto-update from', async function () {
+    if (process.env.TEST_NAME !== 'AUTO_UPDATE_FROM') {
```

Comment:
Open to ideas for a better name for this env var. I think we'll also have AUTO_UPDATE_TO once we add that test next to this one. These tests kinda have to be the only test you run at a time because they replace the test subject.